### PR TITLE
feat: ingest record is annotated with the given annotation key value pair

### PR DESCRIPTION
**Pull request by baranbartu**

### CODE REVIEW

File: cli/cli/scripts/deploy-ray-and-syntho-stack.sh [hunk 1]
```diff
@@ -424,6 +427,30 @@ get_all_logs() {
...
+    SYNTHO_CLI_PROCESS_LOGS="$SYNTHO_CLI_PROCESS_DIR/post_deployment.log"
+
+    echo "post_deployment:annotate_ingress has been started" >> $SYNTHO_CLI_PROCESS_LOGS
+    if ! annotate_ingress >> $SYNTHO_CLI_PROCESS_LOGS 2>&1; then
+        errors+="Annoting ingress has been unexpectedly failed.\n"
+    fi
+    echo "post_deployment:annotate_ingress has been finalized" >> $SYNTHO_CLI_PROCESS_LOGS
+
+    write_and_exit "$errors" "post_deployment"
+}
```
Comment (Younday) [inline cli/cli/scripts/deploy-ray-and-syntho-stack.sh:441]: Just curious: why the setup of annotating the deployment afterwards? We can also just include the annotation in the values for the Helm chart right?

Comment (baranbartu) [reply cli/cli/scripts/deploy-ray-and-syntho-stack.sh:441]: yes we could do it too. I think I just picked the quickest way. Would you prefer that over this?

Comment (baranbartu) [reply cli/cli/scripts/deploy-ray-and-syntho-stack.sh:441]: I noticed [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/templates/frontend/frontend-ingress.yaml#L24-L26)  and [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/values.yaml#L82-L91) and IMO trying to add that annonation selectively to directly ingress would require a bit refactoring there.

Comment (baranbartu) [reply cli/cli/scripts/deploy-ray-and-syntho-stack.sh:441]: mm I think we could add it right after [this](https://github.com/syntho-ai/deployment-tools/blob/a0d955c7eea2c6ffa1ccdf96229c7326e1c0be09/helm/syntho-ui/templates/frontend/frontend-ingress.yaml#L25)  line with a if condition 🤔

**Review by Younday — APPROVED** [review]

File: cli/cli/scripts/deploy-ray-and-syntho-stack.sh (hunk 1 above)
Comment (Younday) [reply cli/cli/scripts/deploy-ray-and-syntho-stack.sh:441]: It's fine to keep it for now as is, in the future we might want to adjust this indeed

**Pull request merged**